This patch causes rsync to report which files caused the unpack_smb_acl()
error, instead of just noting that one happened. This allows the end
user to check the ACL of the reported file by hand.

https://github.com/WayneD/rsync/commit/81c5c81381d86d84fca3a2681ad0c2027bc65d6e
--- a/acls.c	2020-06-25 22:54:21.000000000 -0500
+++ b/acls.c	2020-07-06 16:31:26.000000000 -0500
@@ -519,6 +519,7 @@
 
 		sys_acl_free_acl(sacl);
 		if (!ok) {
+			rsyserr(FERROR_XFER, errno, "get_acl: unpack_smb_acl(%s)", fname);
 			return -1;
 		}
 	} else if (no_acl_syscall_error(errno)) {
